### DISCUSSION TITLE
RESOLVED: was Downtime: Non-"Any" Genre Directories are off air

### DISCUSSION THREAD

**Discussion by pljones** [jamulussoftware/jamulus#2879] [Announcements]
Hi everyone and apologies,

My computer hosting the non-"Any" Genre Directories has had a failed upgrade and is currently not on the network (at all - direct console log in only!!).

I've no idea when I'll have it recovered.

**Comment by ann0see** [comment]
Maintenance: Pinned. Please unpin as soon as the error got resolved.

**Reply by pljones** [reply]
Hopefully working - Jamuluz Explorer looks good.
